### DISCUSSION TITLE
Hydration children mismatch/Conflict between native element and component

### DISCUSSION THREAD

**Discussion by ilyaDegtyarenko** [nuxt/nuxt#17383] [Questions]
I use component names based on its own path directory and filename
https://v3.nuxtjs.org/guide/directory-structure/components#component-names

I have **Table.vue** component in folder **_components/app/Table.vue_**, it used as **< AppTable />**,
but when I want to use a native **< table >< /table >**, it causes a conflict between the native element and the component. One is rendered on the server, another on the client.

```
Hydration children mismatch in < table >: server rendered element contains fewer child nodes than client vdom. 
  at < Table ...
```

Is it possible to force the use of a native element (**< table >**) so as not to change the name of the component (**Table.vue**)?

**Comment by Flanker72** [comment]
It's highly recommended **not** to use reserved names for custom components.
Modern Vue ESlint rules will detect this as an error: https://eslint.vuejs.org/rules/no-reserved-component-names.html

**Reply by ilyaDegtyarenko** [reply]
Can you tell me how I can configure it in Nuxt3?

**Reply by pavloskafritsas** [reply]
> Can you tell me how I can configure it in Nuxt3?

You should add a prefix to the name of your custom components like other frontend libraries do to avoid conflicts with native and other third party libraries (if using any).

Ex: Vuetify components start with V, bootstrap components start with B, etc
